site/app/controllers/grading/ElectronicGraderController.php

```diff
@@ -811,7 +811,12 @@ public function showDetails($gradeable_id, $view = null, $sort = "id", $directio
             $this->core->addErrorMessage("You do not have permission to grade {$gradeable->getTitle()}");
             $this->core->redirect($this->core->buildCourseUrl());
         }
-
+        $anon_mode = false;
+        if (array_key_exists('anon_mode', $_COOKIE)) {
+            if ($_COOKIE['anon_mode'] === 'on') {
+                $anon_mode = true;
+            }
+        }
```

Comment:
We can make this a bit more succinct:
```suggestion
        $anon_mode = isset($_COOKIE['anon_mode']) && $_COOKIE['anon_mode'] === 'on';
```

While `array_key_exists` and `isset` are mostly the same, it's better to use `isset` unless you need some specific functionality of `array_key_exists` as it's a bit faster.